history-files.pl

```diff
@@ -477,74 +498,145 @@
     
     close MAIL;
 }
-
-#
+####################################################################
+# ----------------------------  NLET -------------------------------
 # NLET - Thermally Consequential Non-Load Event Tracking
 #
-# First set the command line to Non-Test
-my $NLET_cmd = "/proj/sot/ska/bin/python /data/acis/LoadReviews/script/NONLOADEVENTTRACKER/RecordNonLoadEvent.py ";
+#
+#   The allowable history-file calls are:   
+#
+#       history-files.pl -stop {time} {status-array} 
+#       history-files.pl -man 
+#       history-files.pl -s107 {time} {status-array}
+#       history-files.pl -too {time} {status array}
+#       history-files.pl -go
+##
+#    Example Status Array: 2002:281:01:43:57.095 HRC-S,HETG-OUT,LETG-OUT,2118,OORMPDS,CSELFMT2,ENAB
+#
+####################################################################
+my $NLET_cmd = "/proj/sot/ska/bin/python /data/acis/LoadReviews/script/NONLOADEVENTTRACKER/RecordNonLoadEvent.py $choice --source history_files.pl ";
```

Comment:
This code should be migrated to Ska3 at some point in the not-too-distant future.  The legacy Ska Python 2 is no longer being maintained.

Reply:
Agreed. As a matter of fact several program are being migrated to ska3 and this one is included. But not in this PR.